contrib/cirrus/test.sh

```diff
@@ -23,17 +23,12 @@ then
         validate)
             showrun ooe.sh git remote add upstream "$CIRRUS_REPO_CLONE_URL"
             showrun ooe.sh git remote update
-            if [[ -z "$CIRRUS_PR" ]]
-            then
-                echo "Testing a branch, assumed or based on the $DEST_BRANCH branch from .cirrus.yml"
-                export GITVALIDATE_EPOCH="$(git rev-parse upstream/$DEST_BRANCH)"
-            else  # Testing a PR
-                echo "Testing a PR targeted at the $DEST_BRANCH branch"
```

Comment:
Does this and the prior comment not hold still?

Reply:
They do, I can add them back in, might be helpful.